### DISCUSSION TITLE
No Auto-Home

### DISCUSSION THREAD

**Discussion by taylotr** [Jyers/Marlin#1851] [General Discussion]
Hello,

I have loaded several versions of the JyersUI firmware and when I go to a version greater than 1.3.1, the autohome function disappears.  Any ideas?

**Comment by Cal-LF** [comment]
Not much to go on. I would say you are using the wrong mainboard version. No auto home is a symptom of using the the wrong firmware.

**Reply by Sonicboom247** [reply]
like @Cal-LF said, make sure you are pulling down the version specific to your board, 4.2.2 / 4.2.7 etc

**Reply by taylotr** [reply]
Sorry to not provide much information.  I have a 4.2.2 board (confirmed by removing the bottom plate of the printer).  I have a BLTouch installed.

The version I was trying to load is:  E3V2-BLTouch-5x5-v4.2.2-v2.0.1.bin
The version that I have working (meaning, with autohome visible) is:  E3V2-BLTouch-5x5-v4.2.2-v1.3.1.bin

**Comment by taylotr** [comment]
Does anyone have any thoughts here?  So far, the 1.3.1 bin is pretty reliable, so don't mind staying with it, but would like to understand what is wrong with the 2.0.1.

**Reply by Sonicboom247** [reply]
Did you reset the Eeprom and restart printer?

**Reply by taylotr** [reply]
The one on-board?  No, but I can certainly try that!

**Reply by Cal-LF** [reply]
@taylotr Golden rule is to always reset the printer eprom memory with every change of firmware. Settings are stored in different locations with every firmware. Even from the same developer.

**Reply by taylotr** [reply]
Interesting, thank you for letting me know.  I have updated the firmware many times on this printer without resetting it.  I presume there is a factory reset option on the printer itself?  I will Google and give it a shot (printing something right now, so need to wait until that is complete).

**Comment by taylotr** [comment]
I reset to factory defaults and still no joy.

**Comment by Cal-LF** [comment]
I use the manual mesh version. I have all functions including home all. Maybe you are looking for something else that is not in the later versions.

**Reply by taylotr** [reply]
@Cal-LF Does the Manual Mesh version work with BLTouch?

**Reply by Cal-LF** [reply]
No.

**Comment by taylotr** [comment]
Let me ask this.  In the menu, is "Prepare > Homing > Home All" the same as "Autohome" and it has been relocated from where I was expecting to see it?

**Reply by Cal-LF** [reply]
@taylotr Yes.

**Reply by taylotr** [reply]
Ouch.

Well, isn't that embarrassing.  I saw it long ago but was truly expecting to see autohome in the same place as usual and never asked.  My apologies for not bringing that up sooner.
